client/src/components/content/Content.js

```diff
@@ -1,37 +1,15 @@
 import React, {Component} from 'react'
 import TopPanel from '../topPanel/TopPanel'
 import MessageList from '../message/MessageList'
-import {Loader} from '../ui'
-import {getReportsByTag} from '../../utils/serverApi'
 import './Content.scss'
 
 class Content extends Component {
-  state = {
-    loading: true,
-  }
-
-  async componentDidMount() {
-    const response = await getReportsByTag(this.props.tag)
-    this.setState({...response, loading: false})
-  }
-
   render() {
-    const {tag} = this.props
-    const {emoji, reports, users, channels, loading} = this.state
-
+    const {tag, reportData: {reports, emoji, users, channels}} = this.props
```

Comment:
This component will be re-rendered every 5 minutes, even if there wasn't a change.